obal/data/modules/tito_build.py

```diff
@@ -0,0 +1,57 @@
+"""
+Release a package using tito
+"""
+
+import re
+from subprocess import CalledProcessError
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.tito import tito  # pylint:disable=import-error,no-name-in-module
+
+
+def main():
+    """
+    Release a package using tito
+    """
+    module = AnsibleModule(
+        argument_spec=dict(
+            directory=dict(type='path', required=True),
+            arguments=dict(type='list', required=False),
+            srpm=dict(type='bool', required=False, default=False),
+            offline=dict(type='bool', required=False, default=False),
+            dist=dict(type='str', required=False),
+            scl=dict(rtype='str', equired=False),
+            output=dict(type='path', required=False),
+        )
+    )
+
+    command = ['tito', 'build']
```

Comment:
I thought about factoring out this param parsing code but I can't come up with anything that's more readable